Test-Get-Jira-Issue.ps1

```diff
@@ -0,0 +1,74 @@
+using module "./src/modules/JiraApis.psm1"
+
+<#
+  .SYNOPSIS
+  Gets Jira isssue details.
+
+  .DESCRIPTION
+  Gets Jira isssue details to help determine what fields values the action should accept.
+
+  .PARAMETER issue
+  Jira issue key. i.e. ABC-1234
+
+  .PARAMETER extendhealth-username
+  Your ExtendHealth Jira username.
+
+  .PARAMETER extendhealth-login
+  Your ExtendHealth Jira password.
```

Comment:
It's just a test file, but since it's in a public repo I still think we should avoid using ExtendHealth.